cloudinit/distros/__init__.py

```diff
@@ -147,7 +147,6 @@ class Distro(persistence.CloudInitPickleMixin, metaclass=abc.ABCMeta):
     resolve_conf_fn = "/etc/resolv.conf"
 
     osfamily: str
-    dhcp_client_priority = [dhcp.IscDhclient, dhcp.Dhcpcd, dhcp.Udhcpc]
```

Comment:
> therwise we see the following across pkg upgrade:

Missing something here, I'm not sure what is supposed to be rendered. 

Also, isn't creating the instance variable in `__init__()` (which we do) sufficient to avoid unpickle issues?

Reply:
You are correct, I had meant to remove this comment before submitting the review. __init__ will cover this case and perform the necessary setup before we reference self.dhcp_client_priority in our code flow. I had tried reproducing the failure case I expected above and didn't see such failure. Closing this comment as irrelevant.